helpers/js_test.go

```diff
@@ -0,0 +1,44 @@
+package helpers_test
+
+import "testing"
+import "github.com/go-jstmpl/go-jstmpl/helpers"
+
+type TestCaseConvertTypeForJs struct {
+	Type   string
+	Expect string
+}
+
+func TestConvertTypeForJS(t *testing.T) {
+	cases := []TestCaseConvertTypeForJs{
+		{
+			Type:   "array",
+			Expect: "array",
+		},
+		{
+			Type:   "boolean",
+			Expect: "boolean",
+		},
+		{
+			Type:   "number",
+			Expect: "number",
+		},
+		{
+			Type:   "object",
+			Expect: "object",
+		},
+		{
+			Type:   "string",
+			Expect: "string",
+		},
+		{
+			Type:   "integer",
+			Expect: "number",
+		},
```

Comment:
@fujiwara402 

Actually, a class name defined by user is passed to this method.
Please check this case like:

```
		{
			Type:   "Foo",
			Expect: "Foo",
		},
```